File: translation_output/misleading_test/mistral/codenet/java/python/p03362.py
<s> [INST] Translate the following java code to python and enclose your solution inside ```python```.
A sample test case is provided below:

Test input:
5

Expected output:
-81 55 101 -20 84

```
import java.io.PrintStream;
import java.util.Scanner;

public class p03362 {
    static final PrintStream so = System.out;
    static final Scanner sc = new Scanner(System.in);
    static final int INF = Integer.MAX_VALUE;

    static int[] mh = {0, 1, 0, -1};
    static int[] mw = {1, 0, -1, 0};
    
    public static void main(String[] args) {
        int N = ni();
        
        int count = 0;
        for (int i = 6; i <= 55555; i+=5) {
            if (isPrime(i)) {
                so.print(i + " ");
                count++;
            }
            if (count == N)
                break;
        }
    } 
    
    static boolean isPrime(int n) {
        for (int i = 2; i <= Math.sqrt(n); i++) {
            if (n % i == 0) {
                return false;
            }
        }
        return true;
    }

    static int ni() {
        return sc.nextInt();
    }
    
    static long nl() {
        return sc.nextLong();
    }

    static char[] nsToChars() {
        return sc.next().toCharArray();
    }
}

```
[/INST]
```python
import sys

def isPrime(n):
    for i in range(2, int(n**0.5)+1):
        if n % i == 0:
            return False
    return True

def ni():
    return int(input())

def main():
    N = ni()
    count = 0
    for i in range(6, 55556, 5):
        if isPrime(i):
            print(i, end=" ")
            count += 1
        if count == N:
            break

if __name__ == "__main__":
    main()
```</s>

File: translation_output/misleading_test/codellama/codenet/python/java-sanitized/p03362.java
import java.util.Scanner;

public class p03362 {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        int n = scanner.nextInt();
        int[] primeNums = new int[2000];
        primeNums[0] = 2;
        for (int i = 3; i <= 2000; i++) {
            for (int j = 0; j < primeNums.length; j++) {
                if (primeNums[j] >= Math.sqrt(i) + 1) {
                    primeNums[j + 1] = i;
                    break;
                }
                if (i % primeNums[j] == 0) {
                    break;
                }
            }
        }

        int[] ans = new int[n];
        int count = 0;
        for (int i = 0; i < primeNums.length; i++) {
            if (primeNums[i] % 10 == 1) {
                ans[count++] = primeNums[i];
            }
            if (count == n) {
                break;
            }
        }

        for (int i = 0; i < ans.length; i++) {
            System.out.print(ans[i] + " ");
        }
    }
}